axelrod.sublime-project

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Agggghhh!!! That's because the change to .gitignore is on another branch and I forgot about it